flexget/plugins/input/parameterize.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils.template import render_from_entry, RenderError
+
+log = logging.getLogger('parameterize')
+
+
+class Parameterize(object):
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            # TODO: This should really be input plugin validation, but some format validators don't deal well with Jinja
+            'plugin': {'type': 'object'},
+            'using': {'$ref': '/schema/plugins?phase=input'},
```

Comment:
maybe name this `from`?

Reply:
I'd think renaming the other section would make more sense for 'from', if either of them do. This is the part that generates the parameters, not that does the entry creating. In my head, the config was describing a sentence e.g. 'Parameterize the html plugin, using settings from rss plugin.' Where in this case 'html' would be the 'plugin' and 'rss' would be the 'using' in the config. I agree this all might be a bit technical naming, but I'm not sure what other sorts of namings would make it any simpler yet. It's a sort of technical concept.